src/depWalker.js

```diff
@@ -352,6 +352,12 @@ async function depWalker(manifest, options = Object.create(null)) {
 
     // Apply warnings!
     payload.warnings = applyWarnings(payload.dependencies);
+    if (payload.warnings.length > 0) {
```

Comment:
Here you need to verify that the `verbose` mode is enabled. Because when we use Node secure as API we probably don't want any logs to happen

Reply:
🤔Not sure if it's a good behavior, I know that in non verbose mode, you don't want any log, but in this case, I think it's important to warn the user about those type of package.

But I could add this verification yup.

Reply:
No because in API this not the end user who pull the data (example SlimIO/Security who use Nodesecure to generate a report). And if the program use for example a Spinner or anything else that work with the terminal then any log will broke the CLI stdout.

Reply:
Well, didn't think about that :D, I check if verbose mode is on now ;)